build/manager/Dockerfile

```diff
@@ -17,7 +17,7 @@ ARG TELEMETRY_TOKEN
 COPY . .
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux go build -v -ldflags "-X github.com/kubeshop/kusk-gateway/pkg/analytics.TelemetryToken=$TELEMETRY_TOKEN" -o manager cmd/manager/main.go
+RUN CGO_ENABLED=0 GOOS=linux go build -v -ldflags "-X github.com/kubeshop/kusk-gateway/pkg/analytics.KuskGAMeasurementID=$GA_ID -X github.com/kubeshop/kusk-gateway/pkg/analytics.KuskGAApiSecret=$GA_SECRET" -o manager cmd/manager/main.go
```

Comment:
Do we need to add the version ldflag here?

Reply:
I haven't touched these as we need to repackage everything.
But yes we need need to pass those values